frontend/templates/apply/apply-base.html.jinja

```diff
@@ -0,0 +1,86 @@
+{% extends "fs-base.html.jinja" %}
+
+
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/cookie-banner/macro.html" import govukCookieBanner %}
+{% from "govuk_frontend_jinja/components/skip-link/macro.html" import govukSkipLink %}
+{% from "govuk_frontend_jinja/components/header/macro.html" import govukHeader %}
+{% from "govuk_frontend_jinja/components/phase-banner/macro.html" import govukPhaseBanner %}
+{% from "govuk_frontend_jinja/components/footer/macro.html" import govukFooter %}
+
+{# TODO sort out service name, use existing mechanism for now #}
+{% set serviceName = gettext("Access Funding") if is_error else service_title or get_service_title() %}
+
+{% block pageTitle %}{{serviceName}}{% endblock pageTitle %}
+{% block headIcons %}
+  {{ super() }}
+{% endblock headIcons %}
+
+{% block skipLink %}
+{{ govukSkipLink({
+    "href": "#main-content",
+    "text": "Skip to main content"
+  })
+}}
+{% endblock skipLink %}
+
+{% block header %}
+<meta name="description" content="Access Funding">
+<meta name="keywords" content="Access Funding">
+<meta name="author" content="Ministry of Housing, Communities and Local Government">
+  {{ govukHeader({
+      "homepageUrl": "#",
+      "containerClasses": "app-width-container",
+      "serviceName": serviceName,
+      "serviceUrl": "/monolith",
+    })
+  }}
+{% endblock header %}
+
+{% set mainClasses = "app-main-class" %}
+
+{% block main %}
+  {{ super() }}
+{% endblock main %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+      "tag": {
+        "text": "Beta"
+      },
+      "html": 'This is a new service.'
```

Comment:
Should we have a feedback/support link in here?

Reply:
I was going with matching what we already have